### ISSUE TITLE
first into listview tap screen found a bug?

### ISSUE THREAD

**Issue by crossle** [johannilsson/android-pulltorefresh#51]
I use pulltorefresh on the listview, when the first into listview, I load data, not pull to refresh.., at this time , I touch the listview, the header TAP_TO_REFRESH Inexplicably appear？ 
but，if into the listview the first ,I pull to refresh. and touch screen .the problem is not exist....
can you help me?

**Comment by crossle** [comment]
How do I manually call onRefresh？

**Comment by Trinea** [comment]
no, it,s maybe caused by call adapter.notifyDataSetChanged(); u can call setSelection(1) to sovle it.
u can use like function setSecondPositionVisible in http://code.google.com/p/trinea-android-common/source/browse/trunk/src/com/trinea/common/view/DropDownToRefreshListView.java

**Comment by crossle** [comment]
I have used setSelection(1) ,but failed!
